.changes/1457-pin-attachments.md

```diff
@@ -0,0 +1,3 @@
+- Pins:
+  - [feature]: You can now add attachments to Pins. enhancing the ability to share additional content and resources. The main attachment is also visible from just Pin list directly.
+  - [fix] occasions where create Pins page opened as sidesheet rather than from bottom.
```

Comment:
Can I haz this as a single level list kthxpls?!?
```suggestion
 - [feature]: You can now add attachments to Pins. enhancing the ability to share additional content and resources. The main attachment is also visible from just Pin list directly.
- [fix] occasions where create Pins page opened as sidesheet rather than from bottom.
```

It's easier to format like that in the changelog generation thing...